{{cookiecutter.project_slug}}/config/settings/local.py

```diff
@@ -69,6 +69,8 @@
 
     hostname, _, ips = socket.gethostbyname_ex(socket.gethostname())
     INTERNAL_IPS += [".".join(ip.split(".")[:-1] + ["1"]) for ip in ips]
+    _, _, ips = socket.gethostbyname_ex("node")
+    INTERNAL_IPS.extend(ips)
```

Comment:
The build failed on Docker because it doesn't know about `node` service. I guess it should be conditional to using a JS task runner:

```suggestion
    {%- if cookiecutter.js_task_runner == 'Gulp' %}
    _, _, ips = socket.gethostbyname_ex("node")
    INTERNAL_IPS.extend(ips)
    {%- endif %}
```

Could we add this task runner to the extended docker test in Travis? What I mean by that is changing the `script` part below:

https://github.com/pydanny/cookiecutter-django/blob/219af571b30d4533631844333bcfad6ad9faee4d/.travis.yml#L20-L21

to:

```
script: sh tests/test_docker.sh use_celery=y use_drf=y 
```